autotest/pyscripts/test_gdal_calc.py

```diff
@@ -55,7 +55,7 @@
 
 def check_file(filename_or_ds, checksum, i=None, bnd_idx=1):
     if gdal_calc.is_path_like(filename_or_ds):
-        ds = gdal.Open(filename_or_ds)
+        ds = gdal.Open(os.fspath(filename_or_ds))
```

Comment:
is this going to work with current bindings ?

Reply:
Yes